intermine_boot/docker.py

```diff
@@ -152,90 +142,90 @@ def create_archives(options, env):
     shutil.make_archive(mine_archive, 'zip', root_dir=mine_data_dir)
 
 
-def lsremote(url):
-    remote_refs = {}
-    g = cmd.Git()
-
-    for ref in g.ls_remote(url).split('\n'):
-        hash_ref_list = ref.split('\t')
-        remote_refs[hash_ref_list[1]] = hash_ref_list[0]
-
-    return remote_refs
-
-def generate_version(options, env):
-    if options['im_repo']!="":
-        currhash = lsremote(options['im_repo'])
-        version = options['im_repo']+"--"+options['im_branch']+"--"+currhash['HEAD']
-    else:
-        version = "latest_version"
-
-    version = version.replace("https://github.com/", "")
-    version = version.replace("/",".")
-
-    return version
-
-def upload_archives(options, env, method):
-    if method == 's3':
-        upload_archives_aws(options, env)
-    else:
-        print ('Method not implemented')
-        raise (NotImplementedError)
-
-def download_archives(options, env, method):
-    if method == 's3':
-        download_archives_aws(options, env)
-    else:
-        print ('Method not implemented')
-        raise (NotImplementedError)
-
-def upload_archives_aws(options, env):
-    compose_path = _get_compose_path(options, env)
-    postgres_archive_path = compose_path.parent / 'data' / 'postgres' / env['data_dir'] / 'postgres.zip'
-    solr_archive_path = compose_path.parent / 'data' / 'solr' / env['data_dir'] / 'solr.zip'
-    mine_archive_path = compose_path.parent / 'data' / 'biotestmine' / env['data_dir'] / 'biotestmine.zip'
-
-    version = generate_version(options, env)
-
-    s3 = boto3.client(
-        's3', 
-        aws_access_key_id=AWS_CLOUD_CREDENTIALS['s3']['ACCESS_KEY'], 
-        aws_secret_access_key=AWS_CLOUD_CREDENTIALS['s3']['SECRET_KEY'])
-    bucket = AWS_CLOUD_CREDENTIALS['s3']['S3_BUCKET_NAME']
-    try:
-        s3.upload_file(str(postgres_archive_path), bucket, str(version+'postgres.zip'))
-        s3.upload_file(str(solr_archive_path), bucket, str(version+'solr.zip'))
-        s3.upload_file(str(mine_archive_path), bucket, str(version+'biotestmine.zip'))
-    except ClientError as error:
-        print(error)
-
-def download_archives_aws(options, env):
-    compose_path = _get_compose_path(options, env)
-    data_dir = compose_path.parent / 'data'
-    postgres_data_dir = compose_path.parent / 'data' / 'postgres'
-    solr_data_dir = compose_path.parent / 'data' / 'solr'
-    mine_data_dir = compose_path.parent / 'data' / 'mine'
-    version = generate_version(options, env)
+# def lsremote(url):
```

Comment:
Could you delete it from the file as well? I don't see any reason to leave it commented out.

Reply:
deleted